apps/theme/layouts/forms/DefaultFormLayout/fields/components/FieldLabel.tsx

```diff
@@ -1,11 +1,35 @@
+import * as React from "react";
 import styled from "@emotion/styled";
+import { ReactComponent as Asterisk } from "@material-symbols/svg-400/outlined/emergency.svg";
```

Comment:
Can we use the regular `*` character here? Instead of adding a new dependency to the `theme` package? 

The less deps, the better, especially when it comes to public website-related code (`website` and `theme` packages).

The thing is - we want to keep the `theme` and `website` folders as light as possible. I get that this won't probaby